spark/src/test/scala/org/apache/comet/parquet/ParquetReadSuite.scala

```diff
@@ -1001,7 +1012,7 @@ abstract class ParquetReadSuite extends CometTestBase {
                 Seq(StructField("_1", LongType, false), StructField("_2", DoubleType, false)))
 
             withParquetDataFrame(data, schema = Some(readSchema)) { df =>
-              if (enableSchemaEvolution) {
+              if (enableSchemaEvolution || usingDataFusionParquetExec(conf)) {
```

Comment:
Hmm does this mean that `usingDataFusionParquetExec==true` we always use schema evolution regardless `COMET_SCHEMA_EVOLUTION_ENABLED` This means it will be incompatible with Spark 3.x?

Reply:
I feel we need a TODO comment or an issue ticket to track this.

Reply:
done